README.md

```diff
@@ -11,6 +11,20 @@ This is entry point document for Mobile Engineering Final Project
 - Support light/dark theme
 - Animation Loading
 
+### Additional Requirement for Final Term:
+- Use Dependecies Injection with Dagger or Hilt
+- Implementing Unit Test
+- Implementing Instrument Test 
+
+## Expectation
+We expect participants to start working on this Final Project from day one. We have 2 grading systems, Mid term and Final term. Grader will evaluate these 3 main point below
+- Functionality, all features should be working properly without any bugs
+- Documentation, provide documentation (Code documentation and project documentation)
+- Code Hygiene, follow engineering best practice
+### Additional for Final Term:
+- Concept, Implementing Design Pattern and Solid Principle
```

Comment:
Add more detail would be nice for
-  `Concept` 
- How many design pattern needed at least
- How many principles from SOLID needed